src/sentry_plugins/jira/plugin.py

```diff
@@ -395,6 +395,11 @@ def create_issue(self, request, group, form_data, **kwargs):
                         v = [v]
                     elif schema.get('custom') == JIRA_CUSTOM_FIELD_TYPES.get('textarea'):
                         v = v
+                    elif schema['type'] == 'number':
+                        try:
+                            v = int(v)
+                        except ValueError:
```

Comment:
This could also raise a `TypeError` if `v` were `None` or some other data structure entirely, so unless you're sure, I'd do `except (TypeError, ValueError):`